### ISSUE TITLE
live djs can't connect to stream for more than 5 seconds without getting dropped 

### ISSUE THREAD

**Issue by ghost** [AzuraCast/AzuraCast#5329]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

 AzuraCast • v0.16.0 Stable • Docker • PHP 8.1 

### What happened?

When a DJ tries to connect to the Azuracast encoder, connection is dropped after 5 seconds with the error Cannot start broadcasting (output 1): Error 2 (unable to open the file/invalid server address)

This happens in the current rolling release channel, as well as the stable channel  (was advised to switch back to stable by support team) 

### Relevant log output

```shell
2022/04/30 11:13:04 [lang:3] DJ Source connected! Last authenticated DJ: Zman2022 - [("content-type", "audio/mpeg"), ("icy-name", "This is my server name"), ("icy-genre", "Rock"), ("icy-url", "http://www./%22), ("icy-pub", "1"), ("icy-irc", "N/A"), ("icy-icq", "N/A"), ("icy-aim", "N/A"), ("icy-br", "192")]
2022/04/30 11:13:04 [lang:3] API djon - Sending POST request to 'http://127.0.0.1:9010/api/internal/1/liquidsoap/djon' with body: { "user": "Zman2022" }
2022/04/30 11:13:04 [lang:3] timeout is deprecated in http.* operators, please use timeout_ms
2022/04/30 11:13:04 [lang:3] API djon - Response (200): true
2022/04/30 11:13:04 [input_streamer:3] Decoding...
2022/04/30 11:13:08 [input_streamer:2] Error while reading from client: Connection reset by peer in read()
2022/04/30 11:13:08 [lang:3] API djoff - Sending POST request to 'http://127.0.0.1:9010/api/internal/1/liquidsoap/djoff' with body: { "user": "Zman2022" }
2022/04/30 11:13:08 [lang:3] timeout is deprecated in http.* operators, please use timeout_ms
2022/04/30 11:13:08 [lang:3] API djoff - Response (200): true
```

**Comment by kcrpine** [comment]
I been having the same issue question , what log were you quoting? So I can check mine just to make sure? Thanks.
Kevin Roberts

**Comment by SadisticSurgeon** [comment]
> I been having the same issue question , what log were you quoting? So I can check mine just to make sure? Thanks. Kevin Roberts

The LS log found under Administration/system log ...

**Comment by SC2Mitch** [comment]
Thanks guys, we're looking into this.

**Comment by doctator81** [comment]
We stumbled upon an unlikely fix to this, at least on our installation. Turning  "Broadcaster Recording" off and then on again seemed to fix the problem. Not sure if this fix will work if broadcaster recording is off, you may have to turn it on and then off again, but that's what we stumbled on that worked .. no clue why lol

**Comment by SlvrEagle23** [comment]
@doctator81 I had an idea that this might be the cause of problems...by chance, do you have your station directory mounted to an external source via a Docker Compose override? I've been testing this thoroughly locally, and had narrowed it down to the recording function as well, but if you use native Docker volumes, there's no issue there.

**Comment by doctator81** [comment]
@SlvrEagle23 No, everything is internal storage. We're not having the problem at the present moment because of what we found that "fixed it" .. but that doesn't mean it won't happen again by random chance.
